README.md

```diff
@@ -53,6 +53,24 @@ Add it to your configuration:
 echo "[secret key]" > config/credentials/development.key
 ```
 
+### Add efile resources locally
+
+In development, we need to download the IRS e-file schemas zip manually from S3.
+
+> ℹ️ We avoid storing them in the repo because the IRS asked us nicely to try to limit distribution.
+
+- Go to [Google Docs and download this file](https://drive.google.com/drive/u/0/folders/1ssEXuz5WDrlr9Ng7Ukp6duSksNJtRATa) (ask a teammate if you don't have access)
+- Do not unzip the file
```

Comment:
You might want to say here that you should run `rake setup:unzip_efile_schemas` to allow it to unzip the file.

Reply:
This is what it says below, but people might try to read just 1 section of the doc.

(also hi this is Asheesh on Travis's machine)

Reply:
`bin/setup` also runs `setup:unzip_efile_schemas`, so added more details to the docs.